packages/fxa-auth-server/lib/payments/iap/google-play/user-manager.ts

```diff
@@ -21,27 +21,23 @@ import Container from 'typedi';
 
 import { AuthLogger } from '../../../types';
 import { PurchaseManager } from './purchase-manager';
-import {
-  GOOGLE_PLAY_FORM_OF_PAYMENT,
-  SubscriptionPurchase,
-} from './subscription-purchase';
-import { PurchaseQueryError } from './types/errors';
-import { SkuType } from './types/purchases';
+import { SubscriptionPurchase } from './subscription-purchase';
+import { PurchaseQueryError } from './types';
+import { UserManager as UserManagerBase } from 'fxa-shared/payments/iap/google-play/user-manager';
 
 /*
  * A class that allows looking up purchases registered to a particular user
  */
-export class UserManager {
-  private log: AuthLogger;
+export class UserManager extends UserManagerBase {
   /*
    * This class is intended to be initialized by the library.
    * Library consumer should not initialize this class themselves.
    */
   constructor(
-    private purchasesDbRef: CollectionReference,
-    private purchaseManager: PurchaseManager
+    public purchasesDbRef: CollectionReference,
```

Comment:
Why is this one public?

Reply:
Ah good catch, let me try making that private again.